webui/react/src/hooks/useSettings.ts

```diff
@@ -280,6 +297,34 @@ const useSettings = <T>(config: SettingsConfig, options?: SettingsHookOptions):
     updateSettings(newSettings);
   }, [ config.settings, updateSettings ]);
 
+  const fetchUserSetting = useCallback(async () => {
+
+    if(!user) return;
+    const userSettingResponse = await getUserSetting({ userId: user.id });
```

Comment:
can we get the `user` from the auth on the API side as opposed to passing it as a parameter?

Reply:
I agree, I should do that.